### DISCUSSION TITLE
Progress of the sync engine

### DISCUSSION THREAD

**Discussion by Pagebakers** [acapela/clientdb#13] [General]
Hey, awesome product! I've been looking for something like this.

The project seems still in early stages, but I'm wondering what is the progress of the sync engine and if it can be used already to set up a demo. It'd be great to build a demo with https://saas-ui.dev

**Comment by jjenzz** [comment]
curious about this too @pie6k. is this project still in active development?

**Reply by heikir** [reply]
Hey @jjenzz 👋🏼 We had a chat with @pie6k this morning and neither one of us has the time right now to continue pushing new updates to clientdb. Since the sync engine is quite a large project, it would require a significant time investment to make it reliably work. We are happy to open-source a sync engine that connects clientdb to supabase and/or Hasura if that would be helpful.

**Reply by jjenzz** [reply]
no worries, totally understand :)

> We are happy to open-source a sync engine that connects clientdb to supabase and/or Hasura if that would be helpful.

i would love to see this tho 🙏

**Reply by Pagebakers** [reply]
That's be awesome!
